qa/rpc-tests/wallet_nullifiers.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python2
+# Copyright (c) 2016 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+from time import *
+
+class WalletNullifiersTest (BitcoinTestFramework):
+
+    def run_test (self):
+        # add zaddr to node 0
+        myzaddr0 = self.nodes[0].z_getnewaddress()
+
+        # send node 0 taddr to zaddr to get out of coinbase
+        mytaddr = self.nodes[0].getnewaddress();
+        recipients = []
+        recipients.append({"address":myzaddr0, "amount":10.0})
+        myopid = self.nodes[0].z_sendmany(mytaddr, recipients)
+
+        opids = []
+        opids.append(myopid)
+
+        timeout = 120
+        status = None
+        for x in xrange(1, timeout):
+            results = self.nodes[0].z_getoperationresult(opids)
+            if len(results)==0:
+                sleep(1)
+            else:
+                status = results[0]["status"]
+                assert_equal("success", status)
+                mytxid = results[0]["result"]["txid"]
+                break
+
+        self.nodes[0].generate(1)
+        self.sync_all()
+
+        # add zaddr to node 2
+        myzaddr = self.nodes[2].z_getnewaddress()
+
+        # import node 2 zaddr into node 1
+        myzkey = self.nodes[2].z_exportkey(myzaddr)
+        self.nodes[1].z_importkey(myzkey)
+
+        # encrypt node 1 wallet and wait to terminate
+        self.nodes[1].encryptwallet("test")
+        bitcoind_processes[1].wait()
+
+        # restart node 1
+        self.nodes[1] = start_node(1, self.options.tmpdir)
+        connect_nodes_bi(self.nodes, 0, 1)
+        connect_nodes_bi(self.nodes, 1, 2)
+        self.sync_all()
+
+        # send node 0 zaddr to note 2 zaddr
+        recipients = []
```

Comment:
This block of code could be refactored locally into a method.  There is another PR with similar method, so once both have been merged, when we have time we could refactor again and move into the test framework.  But not necessary for this PR given time constraints.